test/tokenks/Token.test.js

```diff
@@ -0,0 +1,116 @@
+const Token = artifacts.require('TokenMock');
+const assertRevert = require('../../node_modules/@0xcert/ethereum-utils/test/helpers/assertRevert');
+
+contract('erc/Token', (accounts) => {
+  let token;
+  let owner = accounts[0];
+  let recipient = accounts[1];
```

Comment:
user `const` instead of `let` if possible

Reply:
2c8124e